client/src/main/java/org/apache/rocketmq/client/trace/hook/SendMessageTraceHookImpl.java

```diff
@@ -60,7 +60,6 @@ public void sendMessageBefore(SendMessageContext context) {
         traceBean.setStoreHost(context.getBrokerAddr());
         traceBean.setBodyLength(context.getMessage().getBody().length);
         traceBean.setMsgType(context.getMsgType());
-        traceBean.setClientHost(((AsyncTraceDispatcher)localDispatcher).getHostProducer().getmQClientFactory().getClientId());
```

Comment:
why remove this  clienthost？

Reply:
Because the trace message is a normal message, it has BornHost itself, this is the client IP, there is no need to add in the trace content.Simply to display the trace data, assign the BornHost of the message to the clientHost of the traceBean.
The BornHost of a pub Trace message is the IP of the client producer, and the BornHost of a sub Trace message is the IP of the client consumer.